contributor-guide/modules/ROOT/pages/testing/fuzzing.adoc

```diff
@@ -0,0 +1,174 @@
+= Fuzz testing
+
+*What is fuzz testing?* Fuzzing is a testing technique that injects random pieces of data to a software function to uncover crashes and vulnerabilities. It helps improving code security and reliability, since it can trigger edge cases that went unnoticed during unit testing.
```

Comment:
It's worth expanding on this. Even a subsection. It might be worth describing the bugs fuzz testing usually catches (so the relationship with sanitizers also makes sense later) and how it works (and its relationship with coverage). Otherwise, it might sound like random unit tests that are very complex to write.

Reply:
Good intro sentence, agreed it could be expanded.

Perhaps use Title case for all headings:  Fuzz Testing, Verifying the Effectiveness of your Fuzzer, etc.

Reply:
I've created extra "questions" answering what was missing.
Capitalized all headings.